packages/clarity-js/types/core.d.ts

```diff
@@ -51,6 +51,11 @@ export interface Regions {
     [key: string]: string;
 }
 
+export interface Metrics {
+    /* In the following key-value pair, key is the given CSS Selector and value is Metric enum */
+    [key: string]: number;
```

Comment:
>string [](start = 10, length = 6)

Shouldn't this be a number as it's used that way (I think).

Reply:
The comment is valid. In Javascript we can't have keys as numbers; they can only be strings. So I have selector as a key and metric as a value, so we can keep it as number. 

---
In reply to: [595644577](https://github.com/microsoft/clarity/pull/121#discussion_r595644577) [](ancestors = 595644577)